Outlands_Webminister_Handbook.md

```diff
@@ -20,12 +20,18 @@ The kingdom website shall maintain a library of current governing documents and
 
 ### 2.2 Other Internet Infrastructure
 
+The Kingdom Webminister should provide official outlands.org email accounts for kingdom officers. All official correspondence should come from the provided email.
+
 ### 2.3 Organizational
 
 ### 2.4 Limits of Responsibility
 
 ### 2.5 Online Privacy and Data Policies
 
+> Kingdoms may optionally implement their own data policies. If the Kingdom does not have a data policy the Kingdom Webminister shall have a data policy for the data they control (e.g. databases or emails for which the office is responsible)
+
+The Outlands does not currently have a general data policy. As such, data controlled by a webminister may be retained indefinitely.
```

Comment:
There's an as-yet-unreviewed data policy being included in the website rebuild. Fair point that the data policy needs to be a kingdom discussion.